examples/allegro_hand/allegro_hand.cc

```diff
@@ -132,7 +131,7 @@ class AllegroHandExample : public TrajOptExample {
 
     // Add a model of the hand
     std::string sdf_file = idto::FindIDTOResourceOrThrow(
-        "idto/traj_opt/examples/models/allegro_hand.sdf");
+        "examples/models/allegro_hand.sdf");
```

Comment:
in Drake all file paths must start with `drake/`. 
I wonder if we should adopt that style and make them start with `idto`?
That'd also help disambiguate at call site if the model comes from Drake or from Idto?

Reply:
Yeah, that would look cleaner. Not totally sure how to make that happen though, I can look into it